### PR TITLE
Benchmark

### PR DESCRIPTION
**Pull request by halilkocaoz**
#13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +104,21 @@ Grok grok = new Grok("%{ZIPCODE:zipcode}:%{EMAILADDRESS:email}");
 var grokResult = grok.Parse($"122001:Bob.Davis@microsoft.com");
 ```
 
+# Benchmark
```
Comment (Marusyk) [inline README.md:107]: Please don't add these to the README

File: benchmark/ParseBenchmark.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
+using BenchmarkDotNet.Attributes;
+using GrokNet;
+
+namespace Benchmark
+{
+    public class ParseBenchmark
```
Comment (Marusyk) [inline benchmark/ParseBenchmark.cs:6]: Could you please add MemoryDiagnoser? https://benchmarkdotnet.org/articles/configs/diagnosers.html

File: benchmark/Program.cs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+﻿using BenchmarkDotNet.Running;
+
+namespace Benchmark
+{
+    class Program
+    {
+        static void Main(string[] args) => BenchmarkRunner.Run<ParseBenchmark>();
```
Comment (Marusyk) [inline benchmark/Program.cs:7]: It seems we can remove `string[] args`.
Also, please add modifiers to the class and Main method

**Review by Marusyk — CHANGES_REQUESTED** [review]
Small changes required

**Comment by halilkocaoz** [comment]
Thank you for your time to review and feedback.

**Review by Marusyk — APPROVED** [review]

**Comment by Marusyk** [comment]
@halilkocaoz  Many thanks for your contribution

**Pull request merged**